mlflow/server/__init__.py

```diff
@@ -81,7 +82,10 @@ def serve_get_metric_history_bulk():
 # The files are hashed based on source code, so ok to send Cache-Control headers via max_age.
 @app.route(_add_static_prefix("/static-files/<path:path>"))
 def serve_static_file(path):
-    return send_from_directory(STATIC_DIR, path, max_age=2419200)
+    if Version(flask_version) >= Version("2.0"):
```

Comment:
Can we move this to top-level? We can just compute this flag once.

```
IS_FLASK_2 = Version(flask_version) >= Version("2.0")
```

Reply:
Sure thing, just made that change. Though I used `IS_FLASK_V1` as the variable name instead and flipped the if-statement around since I thought it sounded better than `IS_ATLEAST_FLASK_V2` or something like that.